source/common/http/utility.cc

```diff
@@ -186,6 +186,88 @@ bool Utility::isWebSocketUpgradeRequest(const HeaderMap& headers) {
                     Http::Headers::get().UpgradeValues.WebSocket.c_str())));
 }
 
+Grpc::Status::GrpcStatus Utility::httpToGrpcStatus(uint64_t http_response_status) {
```

Comment:
I'd also suggest leaving this function and the one below under `source/common/grpc`.

Reply:
As it is, it is not possible due to circular Bazel dependencies, i.e., `grpc/common` depends on `http:utility`. Would have to make callers of `Utility::sendLocalReply()` replicate the gRPC status conversion.

Reply:
+1 to @htuch, I think you should be able to move it to source/common/grpc:status, then it won't be circular dependencies.

Reply:
OK, adding source/common/grpc/status_lib.